src/main/java/de/adesso/kicker/user/service/StatisticsService.java

```diff
@@ -35,32 +35,45 @@
 
     private static final int INDEX_OFFSET = 1;
 
-    public void updateRanks() {
-        var rankings = getAllRankings();
-        rankings.forEach(ranking -> {
-            int rank = calculateRank(ranking);
-            ranking.setRank(rank);
-        });
-        saveAllRankings(rankings);
+    void updateRanks() {
```

Comment:
Method isn't either private or public

Reply:
It's package private so I can test the method directly.

Reply:
Alright, just wanted to be sure this is wanted :)